dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime.Tests/DateTimeOffsetVsDateTimeTest.cs

```diff
@@ -0,0 +1,62 @@
+using DateTimeOffsetVsDateTime;
+namespace DateTimeOffsetVsDateTime.Tests;
+
+public class DateTimeOffsetVsDateTimeTest
```

Comment:
A lot of these test cases can be simplified into 1 liners, without the variable declaration.